arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/ModuleOperationStackedSet.java

```diff
@@ -90,21 +76,17 @@ public boolean isEmpty() {
   }
 
   public boolean contains(Object o) {
-    return operationsInTransaction().contains(o) || operationsCommitedToTheConflation().contains(o);
+    return operationsInTransactionBundle().contains(o)
+        || operationsCommitedToTheConflation().contains(o);
```

Comment:
We do have a `contains` for `StackedSet`'s but not `StackedList`'s (which explains my previous question.)